tests/test-kernels/src/bin/open_close_remove_file.rs

```diff
@@ -0,0 +1,23 @@
+use std::{
+	fs::{File, read_to_string, remove_file},
+	io::prelude::*,
+};
+
+#[cfg(target_os = "hermit")]
+use hermit as _;
+
+fn main() {
+	println!("Hello from open_close_remove_file!");
+
+	{
+		let mut file = File::create("/root/foo.txt").unwrap();
+		file.write_all(b"Hello, world!").unwrap();
+	}
+	{
```

Comment:
I don't think the second scope is necessary here

Reply:
This should be resolved now.